src/app/add-event-location/add-event-location.component.ts

```diff
@@ -136,49 +151,59 @@ export class AddEventLocationComponent implements OnInit {
 
   /** controls for the MatSelect filter keyword */
   adminLevelOneFilterCtrl: FormControl = new FormControl();
   adminLevelTwoFilterCtrl: FormControl = new FormControl();
   speciesFilterCtrl: FormControl = new FormControl();
   contactFilterCtrl: FormControl = new FormControl();
 
-  filteredAdminLevelOnes: ReplaySubject<AdministrativeLevelOne[]> = new ReplaySubject<AdministrativeLevelOne[]>();
-  filteredAdminLevelTwos: ReplaySubject<AdministrativeLevelTwo[]> = new ReplaySubject<AdministrativeLevelTwo[]>();
+  filteredAdminLevelOnes: ReplaySubject<AdministrativeLevelOne[]> =
+    new ReplaySubject<AdministrativeLevelOne[]>();
+  filteredAdminLevelTwos: ReplaySubject<AdministrativeLevelTwo[]> =
+    new ReplaySubject<AdministrativeLevelTwo[]>();
 
   buildAddEventLocationForm() {
-    this.addEventLocationForm = this.formBuilder.group({
-      event: null,
-      name: '',
-      start_date: null,
-      end_date: null,
-      country: [APP_UTILITIES.DEFAULT_COUNTRY_ID, Validators.required],
-      administrative_level_one: [null, Validators.required],
-      administrative_level_two: [null, Validators.required],
-      latitude: [null, Validators.pattern(this.latitudePattern)],
-      longitude: [null, Validators.pattern(this.longitudePattern)],
-      land_ownership: [null, Validators.required],
-      gnis_name: '',
-      gnis_id: '',
-      site_description: '',
-      history: '',
-      environmental_factors: '',
-      clinical_signs: '',
-      comment: '',
-      // used only to capture event_type from event - not part of eventlocation record
-      event_type: null,
-      new_location_species: this.formBuilder.array([
-        this.initLocationSpecies()
-      ]),
-      new_location_contacts: this.formBuilder.array([
-        // this.initLocationContacts()
-      ])
-    },
+    this.addEventLocationForm = this.formBuilder.group(
       {
-        validator: [this.endDateBeforeStart, this.startDateTodayorEarlierMortalityEvent]
-      });
+        event: null,
+        name: "",
+        start_date: null,
+        end_date: null,
+        country: [APP_UTILITIES.DEFAULT_COUNTRY_ID, Validators.required],
+        administrative_level_one: [null, Validators.required],
+        administrative_level_two: [null, Validators.required],
+        latitude: [null, Validators.pattern(this.latitudePattern)],
+        longitude: [null, Validators.pattern(this.longitudePattern)],
+        land_ownership: [null, Validators.required],
+        gnis_name: "",
+        gnis_id: "",
+        site_description: "",
+        history: "",
+        environmental_factors: "",
+        clinical_signs: "",
+        comment: "",
+        // used only to capture event_type from event - not part of eventlocation record
+        event_type: null,
+        new_location_species: this.formBuilder.array([
+          this.initLocationSpecies(),
+        ]),
+        new_location_contacts: this.formBuilder.array([
+          // this.initLocationContacts()
```

Comment:
is this supposed to be commented out?

Reply:
Nm I see it being called in another function to create the control

Reply:
Yeah, long story with the form creation stuff, but I'd like the comment to stay for now.